index.bs

```diff
@@ -10710,6 +10495,40 @@ property-installation style as namespaces.)
     1.  Return |F|.
 </div>
 
+
+<h5 id="es-default-operations">Default operations</h5>
+
+Only [=regular operations=] which have a <dfn>corresponding default operation</dfn> defined below
+may be declared with a [{{Default}}] [=extended attribute=].
+
+
+<h6 id="es-default-tojson">default toJSON operation</h5>
+
+The [=corresponding default operation=] of the “toJSON” operation is the [=default toJSON operation=].
+
+<div algorithm>
+
+    To invoke the <dfn export>default toJSON operation</dfn> of [=interface=] |I|,
+    run the the following steps:
+
+    1.  Let |O| be the <emu-val>this</emu-val> value.
+    1.  Let |result| be [=!=] [=ObjectCreate=]([=%ObjectPrototype%=]).
+    1.  If there is an [=inherited interface=] of |I|
+        that declares a “toJSON” operation, then
+        1.  Let |super| be the “toJSON” operation  of
+            the closest [=inherited interface=] of |I|
+            that declares a “toJSON” operation.
+        1.  Set |result| to [=?=] [=Call=](|super|, |O|).
+    1.  If Type(|result|) is not Object, return |result|.
```

Comment:
This behavior allows to set a [Default] extended attribute on a toJSON operation even it has ancestors that set a custom one.

If the custom toJSON operation defined on one of the ancestors returns an object, we just add the interface's attributes to it. Otherwise, we return whatever was returned by the ancestor's toJSON method.

Happy to hear comments as to why this is a bad idea.